src/components/step/HtmlStep/HtmlStep.tsx

```diff
@@ -33,8 +32,7 @@ export default function (props: HtmlStepProps) {
     const customStyleVars = makeCustomStyleVars(props.styles);
 
     return (
-        <>
-            <EmotionGlobal styles={elementsCSS} />
+        <div className="mdhui-html-step">
```

Comment:
There's already a div in InnerHtml.tsx, and this className should work just as well there. To avoid having unneeded nested divs.